transactions/reset-epoch/2024/sept-25/arguments.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "type":"UInt64",
```

Comment:
updated args as per https://github.com/onflow/service-account/pull/330#issuecomment-2374663412 (these args were derived from the output of the network upgrade steps on the ansible box)